samples/subsys/usb/cdc_acm/prj.conf

```diff
@@ -15,5 +15,9 @@ CONFIG_USB_COMPOSITE_DEVICE=n
 
 CONFIG_USB_MASS_STORAGE=n
 CONFIG_SYS_LOG_USB_MASS_STORAGE_LEVEL=1
+
+CONFIG_USB_DEVICE_VID=0xDEAD
```

Comment:
This should not be necessary (L19,L20) because you have set default in `subsys/usb/Kconfig`.

Reply:
Fixed.